### PR TITLE
Makes CAT child pages expandable

### LINKED ISSUE #595: [BUG] CAT API child pages not expandable
### What is the bug?

Page `title` of the CAT API has been changed recently (see [#af4afc2](https://github.com/opensearch-project/documentation-website/commit/af4afc2c1cc09675b9fbdf396beb394ba1554ee5) and [#e669674](https://github.com/opensearch-project/documentation-website/commit/e669674d6419517c772b9dd12d57d1ebe386acd7)).

This means that without relevant changes all child pages become non visible when the CAT API section is expanded.

### How can one reproduce the bug?

Click the CAT API section (as of writing apply to 2.x doc only).
Visit https://opensearch.org/docs/2.0/opensearch/rest-api/cat/index/

### What is the expected behavior?

The value of `parent` field in all child pages should match the value of `title` field in the parent (i.e. in the index.md) file.

For example, the following is excerpt from one of the children:
```
---
layout: default
title: cat field data
parent: CAT
grand_parent: REST API reference
nav_order: 15
has_children: false
---
```

And this is the excerpt from the parent (index.md) file:
```
---
layout: default
title: CAT API
parent: REST API reference
nav_order: 100
has_children: true
redirect_from:
  - /opensearch/catapis/
---
```

`parent` in the child does not match `title` in the parent: `"CAT" != "CAT API"`

### What is your host/environment?

Documentation for 2.x

### Do you have any screenshots?

Screenshot 2022-05-21 at 9 38 57

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Fixes #595 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by alicejw-aws — APPROVED** [review]
Looks good!

File: _opensearch/rest-api/cat/cat-cluster_manager.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ nav_order: 30
 has_children: false
 ---
 
-# CAT cluster_manager
+# cat cluster_manager
```
Comment (alicejw-aws) [inline _opensearch/rest-api/cat/cat-cluster_manager.md:10]: should we keep the CAT acronym in uppercase? 
then we could also update the other CAT API titles under this index to keep it consistent: 
https://opensearch.org/docs/latest/opensearch/rest-api/cat/index/

Comment (alicejw-aws) [inline _opensearch/rest-api/cat/cat-cluster_manager.md:10]: should we keep the CAT acronym in uppercase? 
then we could also update the other CAT API titles under this index to keep it consistent: 
https://opensearch.org/docs/latest/opensearch/rest-api/cat/index/

Comment (kolchfa-aws) [reply _opensearch/rest-api/cat/cat-cluster_manager.md:10]: It was decided to keep the subpages' acronyms lowercase, and the main page's acronym uppercase.

**Review by hdhalter — APPROVED** [review]
Nice work, Fanit!

**Pull request merged**